Framework/src/TaskRunner.cxx

```diff
@@ -142,16 +166,16 @@ CompletionPolicy::CompletionOp TaskRunner::completionPolicyCallback(o2::framewor
     }
   }
 
-  LOG(DEBUG) << "Completion policy callback. "
-             << "Total inputs possible: " << inputs.size()
-             << ", data inputs: " << dataInputsPresent
-             << ", timer inputs: " << (action == CompletionPolicy::CompletionOp::Consume);
+  ILOG(Debug) << "Completion policy callback. "
```

Comment:
I would prefer if we kept this debug log as it was. We don't have a way to suppress infologger debug logs in DPL and this specific one will occur a lot. As soon as this is fixed, we can use infologger interface here.